wls-gui-wahllokalsystem/src/components/wahlvorbereitung/BaseWahlschliessungCard.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-card>
+    <v-card-title>Wahlscheine und Uhrzeit Wahlende</v-card-title>
+    <v-card-text class="pb-0">
+      Bitte geben Sie hier die Uhrzeit ein, zu der die Wahl für geschlossen
+      erklärt wurde.
+      <v-form v-model="schliessungsuhrzeitValidForm">
+        <base-time-input v-model="schliessungsuhrzeit" />
+      </v-form>
+    </v-card-text>
+    <v-card-actions>
+      <base-button-save
+        color="primary"
+        :disabled="isSaveButtonDisabled"
+        @click="onSaveSchliessungsuhrzeitClicked"
+      />
+    </v-card-actions>
+  </v-card>
+</template>
+
+<script setup lang="ts">
+import type { Ref } from "vue";
+
+import { computed, ref } from "vue";
+import {
+  VCard,
+  VCardActions,
+  VCardText,
+  VCardTitle,
+  VForm,
+} from "vuetify/components";
+
+import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import BaseTimeInput from "@/components/common/inputs/BaseTimeInput.vue";
+import { useWahlbezirkStore } from "@/stores/wahlbezirkStore.ts";
+
+const schliessungsuhrzeitValidForm: Ref<null | boolean> = ref(null);
+const schliessungsuhrzeit: Ref<Date | undefined> = ref(undefined);
```

Comment:
Ich finde die vorgeschlagene Schreibweise besser da sie kompakter ist. Ich weiß dass es an einigen Stellen, z.b. im Admintool, von mir anderes gemacht wurde.

```suggestion
const schliessungsuhrzeitValidForm = ref<null | boolean>(null);
const schliessungsuhrzeit = ref<Date | undefined>(undefined);
```

Reply:
done [137fb1e](https://github.com/it-at-m/Wahllokalsystem/pull/1241/commits/137fb1eccbe604517bfaab63f375731f3cdb18d4)